controllers/hosted.go

```diff
@@ -334,32 +332,36 @@ func (r *MultiClusterHubReconciler) calculateHostedStatus(m *operatorv1.MultiClu
 
 	// Copy conditions one by one to not affect original object
```

Comment:
Did you double check this is still true?

Reply:
Yes, I can confirm that this is still true.